app/controllers/cart_controller.rb

```diff
@@ -35,9 +35,10 @@ def remove
  
   def pay
     
-    @cart.orderables.update(discount: @cart.apply_discount)
-    @cart.orderables.update(status: 'paid')
-    @cart.orderables.update(updated_at: @cart.updated_at + 5.days)
+    @cart.orderables.update(discount: @cart.apply_discount,status: 'paid',updated_at: @cart.updated_at + 5.days)
```

Comment:
whats the purpose of updating updated_at attribute to 5+days?

Reply:
to update delivery date i'm using updated_at as delivery date so 5+ is the day spans